### PR TITLE
LVM-309 Adds fields to student table

### PR DESCRIPTION
**Pull request by vcantu**
[Jira Ticket](https://cs5500-jira.ccs.neu.edu/browse/LVM-309)

[Student From Spec](https://cs5500.ccs.neu.edu/confluence/display/CS4500Sp16TEAM4/S17+LVM+Forms+and+Specifications?preview=/13534286/13534280/2013%20Basic%20Literacy%20Intake%20Form%20-%20LVM%20Boston.pdf)

Changes the following from the students table
- -name
- +first_name
- +last_name
- +dob
- +gender
- +address
- +city
- +state
- +zip
- +mail_ok
- +email
- +email_ok
- +cell_phone
- +cell_ok
- +cell_lvm_ok
- +home_phone
- +home_ok
- +home_lvm_ok
- +work_number
- +work_ok
- +work_lvm_ok
- +emergency_name
- +emergency_number
- +referral
- +why_lvm
- +race
- +is_hispanic
- +native_language
- +origin_country
- +availability
- +tutor_preference

### CODE REVIEW

File: coverage/.last_run.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "result": {
+    "covered_percent": 94.15
+  }
+}
```
Comment (zack) [inline coverage/.last_run.json:5]: Did you add these coverage files explicitly? The `.gitignore` file should be ignoring them.

**Comment by zack** [comment]
I think you need to rebase!

`git fetch upstream`
`git rebase upstream/master`
`git push origin -f`

File: coverage/.last_run.json (hunk 1 above)
Comment (vcantu) [reply coverage/.last_run.json:5]: i did not add them

Comment (zack) [reply coverage/.last_run.json:5]: That's very weird. Would you mind running

`rm coverage/.last_run.json`
and 
`rm .resulttest.json`

**Comment by zack** [comment]
Okay, you've just got some conflicts now with my most recent changes to the student spec. No big deal.

Find line 231 in the student spec and remove all of the =====s and stuff so it looks like this;

```ruby
            it 'does not set up a new match for the student' do
              put :set_tutor, params: {
                tutor_id: @new_tutor, student_id: @student
              }
              # That is, the student started with and ended with exactly 1 match
              expect(Match.where(student_id: @student).length).to eq(1)
            end
```

File: app/models/student.rb [hunk 2]
```diff
@@ -5,6 +5,5 @@ class Student < ApplicationRecord
   has_many :enrollments
   has_many :coordinators, through: :enrollments
 
-  validates :name, presence: true,
-                   length: { minimum: 3 }
+  validates :first_name, presence: true, length: { minimum: 3 }
```
Comment (zack) [inline app/models/student.rb:8]: I think we can probably remove the length requirement.

**Pull request merged**